cli/console.ts

```diff
@@ -331,6 +331,25 @@ export function printExecutedAction(
   }
 }
 
+export function printFormatFilesResult(
+  formatResults: Array<{
+    filename: string;
+    err?: Error;
+  }>
+) {
+  const sorted = formatResults.sort((a, b) => a.filename.localeCompare(b.filename));
+  const successful = sorted.filter(result => !result.err);
+  const failed = sorted.filter(result => !!result.err);
+  if (successful.length > 0) {
+    printSuccess("Successfully formatted:");
+    successful.map(result => result.filename).forEach(filename => writeStdOut(filename, 1));
```

Comment:
it seems the map method is redundant, you can do same stuff under forEach, like line 349.